### DISCUSSION TITLE
Sobre archivo main.c y makefile

### DISCUSSION THREAD

**Discussion by ManuelV03** [IIC2133-PUC/2024-1#22] [Tarea 0] [answered]
Hola, me entró la duda si en el archivo main.c que viene con nuestros repositorios hay que dejar intacta la parte que viene prehecha, o si puedo modificarle cosas como el nombre a las variables u otras cosas. Una duda similar me surge con el makefile, para este, si añadimos más archivos a nuestra tarea que deseamos compilar hay que escribirlos dentro o el makefile que nos dan por default se encarga de esto?

**Comment by CataAlamos** [comment, marked as answer]
Hola! Si quieren crear nuevos archivos debes importarlos en main.c y este puedes modificarlo todo lo que quieras, te sugiero no cambiar el código que les damos pero eso queda a tu criterio. En cuanto al makefile no cambies, esta configurado para que puedan realizar la tarea.

**Reply by ManuelV03** [reply]
Gracias!

**Reply by CataAlamos** [reply]
De nada, éxito en la tarea!
